README.md

```diff
@@ -8,18 +8,77 @@
 
-1. Generating your "Cut A Release" Github CI for you
-2. Picking good build flags for a "production" release
-3. Making zips and installers for the resulting binaries
-4. Generating a machine-readable manifest so other tools can understand the results
-5. Uploading all the resulting artifacts to a Github Release™️
+```sh
+git commit -am "release: 0.2.0"
+git tag "v0.2.0"
+git push
+git push --tags
+```
+
+Will make [this Github Release](https://github.com/axodotdev/axolotlsay/releases/tag/v0.2.0):
+
+Or if you're using [oranda](https://opensource.axo.dev/oranda/), you'll get [this website](https://opensource.axo.dev/axolotlsay/).
+
+
+## Plan, Build, Host, Publish, Announce
+
+Cutting releases of your apps and distributing binaries for them has a lot of steps, and cargo-dist is quickly growing to try to cover them all!
+
+To accomplish this, cargo-dist functionality can be broken up into two parts:
+
+* building (**planning** the release; **building** binaries and installers)
+* distributing (**hosting** artifacts; **publishing** packages; **announcing** releases)
+
+The build functionality can be used on its own if you just want some tarballs and installers, but everything really comes together when you use the distribution functionality too.
+
+
+## Building
+
+As a build tool, cargo-dist can do the following:
+
+* Pick good build flags for "shippable binaries"
+* Make [tarballs][] and [installers][] for the resulting binaries
+* Generate [machine-readable manifests][manifest] so other tools can understand the results
+
+That's a short list because "we make [installers][]" is doing a lot of heavy lifting. Each installer could be (and sometimes is!) an entire standalone tool with its own documentation and ecosystem.
+
+
+## Distributing
+
+As a distribution tool, cargo-dist gets to flex its biggest superpower: **it generates its own CI scripts**. All you need to do is enable the GitHub CI backend, and `cargo dist init` will generate release.yml, which implements the full pipeline of plan, build, host, publish, announce:
+
+* Plan
+    * Waits for you to push a git tag for a new version (v1.0.0, my-app-v1.0.0, my-app/1.0.0, ...)
+    * Selects what apps in your workspace to announce new releases for based on that tag
+    * Generates [a machine-readable manifest][manifest] with changelogs and build plans
+* Build
+    * Spins up machines for each platform you support
+    * Builds your [binaries and tarballs][tarballs]
+    * Builds [installers][] for your binaries
+* Publish:
+    * Uploads to package managers
+* Host + Announce:
+    * Creates (or edits) a GitHub Release
+    * Uploads build artifacts to the Release
+    * Adds relevant release notes from your RELEASES/CHANGELOG
+
+(Ideally "host" would come cleanly before "publish", but GitHub Releases doesn't really properly support this kind of staging, so we're forced to race the steps a bit here. Future work may provide a more robust release process.)
```

Comment:
I think this is too much detail to introduce at this stage of the README. People are still just reading about what we do.